src/references/eval.md

```diff
@@ -0,0 +1,58 @@
+# Eval 
+
+Each AO process includes an onboard `Eval` Handler, which evaluates any new code sent to it. This Handler enables the process to determine the appropriate action for code being added and checks if the message came from the process owner.
+
+The `Eval` handler can also be manually triggered to evaluate received data, such as when a parent process sends a message to upload new handlers to its child process.
+
+## Sending The Eval Action To Load Code Into A Child Process
+```lua
+Send({
+    Target = Child_Process,
+    Action = "Eval",
+    Data = [[
+        Handlers.add("ping", Handlers.utils.reply("pong"))
+    ]]
+})
+```
+
+## Eval Handler (_eval)
```

Comment:
I don't think it is needed to show the implementation of the eval function for the guide, 

I would just show how to call eval from javascript/typescript